### PR TITLE
Italian revision

### PR DESCRIPTION
**Pull request by Alby1987**

### CODE REVIEW

**Comment by NeonKnightOA** [comment]
Excellent! Let me review it before committing it.

As a starting note, bear in mind that changes to the following UT files must also be placed in their Unreal versions and viceversa, for the sake of consistency:

* **ALAudio.itt**
* **Core.itt**
* **D3D9Drv.itt**
* **D3DDrv.itt**
* **Engine.itt**
* **FMODAudioDrv.itt**
* **Galaxy.itt**
* **GlideDrv.itt**
* **IpDrv.itt**
* **IpServer.itt**
* **MeTaLDrv.itt**
* **OpenGLDrv.itt**
* **Setup.itt**
* **SglDrv.itt**
* **SoftDrv.itt**
* **Startup.itt**
* **UBrowser.itt**
* **UMenu.itt** (and UT's **UTMenu.itt**)
* **UnrealEd.itt**
* **UnrealI.itt**
* **UnrealShare.itt**
* **UWindow.itt**
* **Window.itt**
* **WinDrv.itt**
* **XMesaGLDrv.itt**
* **XOpenGLDrv.itt**

File: UnrealTournament/SystemLocalized/itt/Botpack.itt [hunk 1]
```diff
@@ -3107,11 +3108,14 @@ ItemName="Caricatore"
 ItemName="Enforcer Doppi"
 
 [Translocator]
-; EN: WeaponDescription="Classification: Personal Teleportation Device\n\nPrimary Fire: Launches the destination module.  Throw the module to the location you would like to teleport to.\n\nSecondary Fire: Activates the translocator and teleports the user to the destination module.\n\nTechniques: Throw your destination module at another player and then activate the secondary fire, and you will telefrag your opponent!  If you press your primary fire button when activating your translocator with the secondary fire, the last weapon you had selected will automatically return once you have translocated."
```
Comment (NeonKnightOA) [inline UnrealTournament/SystemLocalized/itt/Botpack.itt:3110]: I'm curious, why the change of line here?

File: UnrealTournament/SystemLocalized/itt/DM-HealPod][.itt [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ﻿[LevelInfo0]
 ; EN: Title="Healing Pod ]["
-Title="Baccello di Guarigione ]["
+Title="Piattaforma di Guarigione ]["
 IdealPlayerCount="2-8"
```
Comment (NeonKnightOA) [inline UnrealTournament/SystemLocalized/itt/DM-HealPod][.itt:4]: Remember to also bring these changes to DmHealPod.itt in Unreal.

File: UnrealTournament/SystemLocalized/itt/DM-Cybrosis][.itt [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ﻿[LevelInfo0]
 ; EN: Title="Cybrosis Central ]["
-Title="Central Cybrosis ]["
+Title="Centrale Cybrosis ]["
```
Comment (NeonKnightOA) [inline UnrealTournament/SystemLocalized/itt/DM-Cybrosis][.itt:3]: Remember to also bring these changes to DM-Cybrosis.itt in Unreal.

File: UnrealTournament/SystemLocalized/itt/Engine.itt [hunk 4]
```diff
@@ -25,7 +25,7 @@ NameArticle=" un "
 
 [Inventory]
 ; EN: PickupMessage="Snagged an item"
-PickupMessage="Liberato un oggetto"
+PickupMessage="Recuperato un oggetto"
```
Comment (NeonKnightOA) [inline UnrealTournament/SystemLocalized/itt/Engine.itt:28]: Remember to also check Engine.itt in Unreal and sync your changes there too.

File: UnrealTournament/SystemLocalized/itt/GlideDrv.itt [hunk 5]
```diff
@@ -8,4 +8,4 @@ ClassCaption="3dfx Glide"
 ; EN: AskInstalled="Do you have a 3dfx card installed?"
 AskInstalled="Hai una scheda 3dfx installata?"
 ; EN: AskUse="Unreal Tournament is compatible with 3dfx Voodoo 1 and Voodoo 2, but not with Voodoo Rush-based cards. Do you want Unreal Tournament to use your 3dfx card?"
-AskUse="Unreal Tournament è compatibile con 3dfx Voodoo 1 e Voodoo 2, non lo è con le schede basate su Voodoo Rush. Vuoi che Unreal Tournament utilizzi la tua carta 3dfx?"
+AskUse="Unreal Tournament è compatibile con 3dfx Voodoo 1 e Voodoo 2, non lo è con le schede basate su Voodoo Rush. Vuoi che Unreal Tournament utilizzi la tua scheda 3dfx?"
```
Comment (NeonKnightOA) [inline UnrealTournament/SystemLocalized/itt/GlideDrv.itt:11]: Remember to also check GlideDrv.itt in Unreal and sync your changes there too.

File: UnrealTournament/SystemLocalized/itt/IpDrv.itt [hunk 6]
```diff
@@ -60,4 +60,4 @@ ClassCaption="Gioco in rete TCP/IP"
 
 [UdpBeacon]
 ; EN: ClassCaption="Server LAN Beacon"
-ClassCaption="Server Beacon LAN"
+ClassCaption="Server LAN Beacon"
```
Comment (NeonKnightOA) [inline UnrealTournament/SystemLocalized/itt/IpDrv.itt:63]: Remember to also check IpDrv.itt in Unreal and sync your changes there too.

File: UnrealTournament/SystemLocalized/itt/IpServer.itt [hunk 7]
```diff
@@ -8,4 +8,4 @@ ClassCaption="Connessione Server principale"
 
 [ServerQuery]
 ; EN: ClassCaption="Game Server Query"
-ClassCaption="Query al Server del gioco"
+ClassCaption="Query al Server del Gioco"
```
Comment (NeonKnightOA) [inline UnrealTournament/SystemLocalized/itt/IpServer.itt:11]: Remember to also check IpServer.itt in Unreal and sync your changes there too.

File: UnrealTournament/SystemLocalized/itt/MeTaLDrv.itt [hunk 8]
```diff
@@ -6,6 +6,6 @@ Preferences=(Caption="S3 MeTaL",Parent="Rendering",Class=MeTaLDrv.MeTaLRenderDev
 ; EN: ClassCaption="S3 MeTaL"
 ClassCaption="S3 MeTaL"
 ; EN: AskInstalled="Do you have a Savage 3D card installed?"
-AskInstalled="Hai una scheda video Savage 3D card installata?"
+AskInstalled="Hai una scheda video Savage 3D installata?"
```
Comment (NeonKnightOA) [inline UnrealTournament/SystemLocalized/itt/MeTaLDrv.itt:9]: Remember to also check MeTaLDrv.itt in Unreal and sync your changes there too.

File: UnrealTournament/SystemLocalized/itt/OpenGLDrv.itt [hunk 9]
```diff
@@ -8,7 +8,7 @@ NoFindGL="Non riesco a trovare il driver OpenGL %ls"
 ; EN: MissingFunc="Missing OpenGL function %ls (%i)"
 MissingFunc="Manca la funzione OpenGL %ls (%i)"
 ; EN: ResFailed="Failed to set resolution"
-ResFailed="Errore nel settare la risoluzione"
+ResFailed="Errore nell'impostare la risoluzione"
```
Comment (NeonKnightOA) [inline UnrealTournament/SystemLocalized/itt/OpenGLDrv.itt:11]: Remember to also check OpenGLDrv.itt and XOpenGLDrv.itt in Unreal and sync your changes there too.

File: UnrealTournament/SystemLocalized/itt/SDLDrv.itt [hunk 10]
```diff
@@ -23,8 +23,8 @@ Color16="Colori a &16-bit"
 ; EN: Color32="&32-bit color"
 Color32="Colori a &32-bit"
 ; EN: AdvancedOptions="Advanced Options"
-AdvancedOptions="Opzioni Avanzate"
+AdvancedOptions="Advanced Options"
```
Comment (NeonKnightOA) [inline UnrealTournament/SystemLocalized/itt/SDLDrv.itt:26]: Remember to also check SDL2Drv.itt in Unreal and sync your changes there too.

File: UnrealTournament/SystemLocalized/itt/Setup.itt [hunk 11]
```diff
@@ -218,4 +218,4 @@ IDC_CompleteReboot="Installazione completata!\Prima di giocare, è necessario ri
 
 [IDDIALOG_WizardDialog]
 ; EN: IDC_WizardDialog="Unreal Tournament Setup"
-IDC_WizardDialog="Archivio Unreal Tournament"
+IDC_WizardDialog="Installazione di Unreal Tournament"
```
Comment (NeonKnightOA) [inline UnrealTournament/SystemLocalized/itt/Setup.itt:221]: Remember to also check Setup.itt and SetupUnrealPatch.itt in Unreal and sync your changes there too.

File: UnrealTournament/SystemLocalized/itt/UBrowser.itt [hunk 12]
```diff
@@ -1,14 +1,14 @@
 ﻿[UBrowserGSpyLink]
 ; EN: ResolveFailedError="The master server could not be resolved: "
-ResolveFailedError="Impossibile raggiungere il server primario: "
+ResolveFailedError="Impossibile raggiungere il server principale: "
```
Comment (NeonKnightOA) [inline UnrealTournament/SystemLocalized/itt/UBrowser.itt:3]: Remember to also check UBrowser.itt in Unreal and sync your changes there too.

File: UnrealTournament/SystemLocalized/itt/UMenu.itt [hunk 13]
```diff
@@ -63,7 +63,7 @@ ServerNameHelp="Inserisci la descrizione completa del tuo server, che apparirà
 ; EN: DoUplinkText="Advertise Server"
 DoUplinkText="Reclamizza Server"
 ; EN: DoUplinkHelp="If checked, your server will be advertised to the Master Server, so your server will appear in the global server list."
-DoUplinkHelp="Se marcata, il tuo server verrà reclamizzato presso il server primario, in modo che gli utenti provvisti di strumenti di interrogazione come UBrowser o GameSpy possano vedere il tuo server."
+DoUplinkHelp="Se marcata, il tuo server verrà reclamizzato presso il server principale, in modo che gli utenti provvisti di strumenti di interrogazione come UBrowser o GameSpy possano vedere il tuo server."
```
Comment (NeonKnightOA) [inline UnrealTournament/SystemLocalized/itt/UMenu.itt:66]: Remember to also check UMenu.itt in Unreal and sync your changes there too.

File: UnrealTournament/SystemLocalized/itt/XOpenGLDrv.itt [hunk 14]
```diff
@@ -9,7 +9,7 @@ NoFindGL="Non riesco a trovare il driver OpenGL %ls"
 ; EN: MissingFunc="Missing OpenGL function %ls (%i)"
 MissingFunc="Manca la funzione OpenGL %ls (%i)"
 ; EN: ResFailed="Failed to set resolution"
-ResFailed="Errore nel settare la risoluzione"
+ResFailed="Errore nell'impostare la risoluzione"
```
Comment (NeonKnightOA) [inline UnrealTournament/SystemLocalized/itt/XOpenGLDrv.itt:12]: Remember to also check OpenGLDrv.itt and XOpenGLDrv.itt in Unreal and sync your changes there too.

**Review by NeonKnightOA — COMMENTED** [review]
Reviewed and left notes.

**Comment by NeonKnightOA** [comment]
Scratch it, I'm committing your changes.
We'll have enough time to fix what's needed.

File: UnrealTournament/SystemLocalized/itt/Botpack.itt (hunk 1 above)
Comment (Alby1987) [reply UnrealTournament/SystemLocalized/itt/Botpack.itt:3110]: This was a really strange bug: in the weapon description window, the translocator had missing title (the english one was shown) and the description was cutted (was in italian but incomplete). Trying to understand why, I discovered that ordering those lines in diverse order in Botpack.itt file, the title was correctly translated, but the description was cutted ever more.

Trying, I put more padding using those comments, and then the Translocator window was perfect: italian title and the whole description shown. This is why I put those padding lines.

Comment (NeonKnightOA) [reply UnrealTournament/SystemLocalized/itt/Botpack.itt:3110]: That's indeed weird. Let me check it out.
